splice_machine/src/main/java/com/splicemachine/derby/stream/control/ControlTableChecker.java

```diff
@@ -157,7 +178,7 @@ private void populateData(PairDataSet table, PairDataSet index) {
 
     private List<String> checkInvalidIndexes() throws StandardException {
```

Comment:
For checkDuplicateIndexes(), shouldn't we also handle the fix=true path, and remove duplicate indexes?

Reply:
This does not happen for system tables. I'll file another Jira to address this issue.